projects/matio/project.yaml

```diff
@@ -0,0 +1,9 @@
+homepage: "https://github.com/tbeu/matio"
+primary_contact: "t-beu@users.sourceforge.net"
```

Comment:
I can't use my SF mail address for a Google account since the dash is not allowed. Can you please use my other address modelica at tbeu dot de where I already have an account. Thanks.

![grafik](https://user-images.githubusercontent.com/5444067/63913157-5a5f9d00-ca2f-11e9-9284-fe662d9b5ede.png)

Reply:
OK. That SF mail address works finally works for me.